next.config.js

```diff
@@ -1,7 +1,15 @@
 const { createSecureHeaders } = require("next-secure-headers");
 
+const secureHeaderOptions = {
+  contentSecurityPolicy: {
+    directives: {
+      frameSrc: ["; frame-ancestors", "'self'", "*"],
+    },
+  }
+};
+
 module.exports = {
   async headers() {
-    return [{ source: "/(.*)", headers: createSecureHeaders() }];
+    return [{ source: "/(.*)", headers: createSecureHeaders(secureHeaderOptions) }];
```

Comment:
Should these headers just apply to the embed view?

Reply:
Yea, that makes total sense, I'll do that